doc/classes/EditorSettings.xml

```diff
@@ -756,6 +756,8 @@
 		<member name="project_manager/sorting_order" type="int" setter="" getter="">
 			The sorting order to use in the project manager. When changing the sorting order in the project manager, this setting is set permanently in the editor settings.
 		</member>
+		<member name="resource_scene_unique_id" type="String" setter="set_scene_unique_id" getter="get_scene_unique_id">
+		</member>
```

Comment:
This is quite odd. I am to assume that the doctools generated this on its own, but it should not happen...

Reply:
You need to add a `_` prefix to prevent it from being documented I think, usage doesn't control that

Reply:
This is because it has its own processing AFAIK

Reply:
Me no likey. Sounds like a separate bug to tackle.

Reply:
It turns out it's an explicit list that needs to be updated

Reply:
You need to remove this